pulsar-functions/api-java/src/main/java/org/apache/pulsar/functions/api/WindowContext.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pulsar.functions.windowing;
+package org.apache.pulsar.functions.api;
```

Comment:
this is an API breaking change. can we document that?

Reply:
@sijie This is not api breaking change. Previously WindowContext was not exposed to the user at all. It was just implementation detail. Now we are exposing this.
I will add up documentation as part of the another pr because there are other changes on top of this.